.eslintrc.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
     'jest/no-disabled-tests': 'error',
     'jest/no-focused-tests': 'error',
     'jest/no-identical-title': 'warn',
-    'linebreak-style': ['error', 'unix'],
+    'linebreak-style': ['error', 'windows'],
```

Comment:
Why are we switching to CRLF? .editorconfig defines Unix line endings for this project

Reply:
I was getting errors in es-lint unless I changed this. my bad.

Reply:
No worries! It might be an issue in the lint setup if correctly-formatted files throw errors on Windows.

Reply:
because this is running on ubuntu we need this to be kept as unix